createdisk-library.sh

```diff
@@ -202,7 +202,7 @@ enabled=1
 gpgcheck=0
 EOF'"
         # Install these rpms to VM
-        ${SSH} core@${vm_ip} -- "sudo rpm-ostree install $ADDITIONAL_PACKAGES"
+        ${SSH} core@${vm_ip} -- "sudo rpm-ostree install $ADDITIONAL_PACKAGES $PRE_DOWNLOAD_ADDITIONAL_PACKAGES"
```

Comment:
Maybe name this `ADDITIONAL_LOCAL_PACKAGES`? If you go with this naming, `ADDITIONAL_PACKAGES` could be renamed to `ADDITIONAL_DNF_PACKAGES`
If you want to keep the current name, this should be `PRE_DOWNLOADED_ADDITIONAL_PACKAGES`

Reply:
>If you want to keep the current name, this should be PRE_DOWNLOADED_ADDITIONAL_PACKAGES

I will go with that and in my mind I had `PRE_DOWNLOADED_..` only :)